webapp/src/Command/CheckDatabaseConfigurationDefaultValuesCommand.php

```diff
@@ -66,11 +66,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
         if (empty($messages)) {
             $style->success('All default values have the correct type');
-        } else {
-            $style->error('Some default values have the wrong type:');
-            $style->listing($messages);
+            return Command::SUCCESS;
         }
-
-        return Command::SUCCESS;
+        $style->error('Some default values have the wrong type:');
+        $style->listing($messages);
+        return Command::FAILURE;
```

Comment:
Somehow it feels 'nicer' that the default outcome is success, so should we negate the if maybe?

Reply:
I agree it feels nicer, but I didn't like the negation, will implement a fix.